tests/integration/test_identify_service.py

```diff
@@ -225,7 +225,7 @@ def test_invalid_imageDisplay(self):
         self.assertTrue(resp.json['detail'].startswith('Please provide the parameter imageDisplay in a comma separated list of 3 arguments (width,height,dpi)'))
 
         resp = self.testapp.get('/rest/services/ech/MapServer/identify', params=params, headers={'Accept': 'text/html'}, status=400)
-        self.assertEqual(resp.content_type, 'text/html')
+        self.assertEqual(resp.content_type, 'application/json')  # All errors are json
```

Comment:
"All errors are json" is this defined globally/on Pyramid level? In the code I cannot find the place where content-type is changed to json when request header has `'accept':'text/html'`...

Reply:
No, the render was set in the decorator of the view `@exception_view_config(context=HTTPException)`. I had to remove it, because only one decorator may be set per view. So returning a string and setting the correct mime-type should be enough.